markets/legacy-market/cannonfile.toml

```diff
@@ -163,14 +154,14 @@ from = "<%= settings.pool_owner %>"
 args = [
     "<%= settings.sc_pool_id %>",
     [
-        { marketId = "1", weightD18 = "1", maxDebtShareValueD18 = "1000000000000000000" },
+        { marketId = "<%= extras.marketId %>", weightD18 = "1", maxDebtShareValueD18 = "1000000000000000000" },
     ]
 ]
-depends = ["invoke.createPool", "invoke.registerMarket"]
+depends = ["invoke.createPool"]
```

Comment:
its actually pretty easy:
1. try to build cannon file
2. cannonfile fails with some sort of error. when this happens, cannon includes a trace which shows you where the failure happened and why. In these cases, you would see an error caused by a previously required step not being completed
3. Add the previously required step to the `depends` array.

So in this case, I ran into an error because the cannonfile had an error because the pool cant be configured before its created. To solve this error, I simply added it as a depends, because cannon has no way of detecting this.

Register market dependency was reoved because the register market step exports the `extras.marketId` property, whhich is used in the interpolation for the step.